tests/test_data.py

```diff
@@ -1,12 +1,12 @@
 import pytest
 from ase.build import bulk, molecule
 
-from zndraw.data import atoms_from_json, atoms_to_json
+from zndraw.frame import Frame
 
 
 @pytest.mark.parametrize(
     "atoms", [(molecule("C6H6")), (molecule("H2O")), (bulk("Cu", "fcc", a=3.6))]
 )
 def test_atoms_from_and_to_json_return_unchanged_atoms_object(atoms):
     atom_copy = atoms.copy()
-    assert atom_copy == atoms_from_json(atoms_to_json(atoms))
+    assert atom_copy == (Frame.from_atoms(atom_copy)).to_atoms()
```

Comment:
I dont really get the second line. I assume you want something along the lines of
`frame = Frame.from_atoms(test_atoms)
assert frame == ??
assert frame frame.to_atoms == test_atoms` 
but what is the second line supposed to be? Because `assert frame == Frame.from_atoms(test_atoms)` would be always satisfied as we declare it a line above.

Reply:
Something like the following

```py
frame = Frame.from_atoms(atom_copy)
assert frame == Frame(positions=atom_copy.position, ...)
assert frame.to_atoms() == atom_copy

```